node/service/src/parachains_db/mod.rs

```diff
@@ -40,6 +42,11 @@ pub(crate) mod columns {
 		pub const ORDERED_COL: &[u32] =
```

Comment:
I'd prefer to move all column data to `v2` and only leave `NUM_COLUMNS` in `v1`. We did the same when introducing `v1`.